### PR TITLE
Feature/mx 1537 extract voxco data

### PR DESCRIPTION
**Pull request by erichesse**
# Added
- extract voxco data 

# Changes
- consolidate mocked drop connector into one general mock

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,12 +33,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - transform grippeweb resources
 - wikidata aux extractor into seq-repo
 - function `get_merged_organization_id_by_query_with_transform_and_load` to
-  wikidata.extract module
+ wikidata.extract module
```
Comment (mr-kamran-ali) [inline CHANGELOG.md:36]:
```suggestion
 - wikidata.extract module
```

File: tests/drop/mocked_drop.py [hunk 2]
```diff
@@ -20,11 +20,27 @@ def mocked_drop_for_seqrepo(monkeypatch: MonkeyPatch) -> None:
...
+                / "tests"
+                / x_system.replace("-", "_")
+                / "test_data"
+            ).rglob("*.json")
+        ],
     )
 
-    def get_file_mocked(*_, **__):
-        with open(Path(__file__).parent / "test_data" / "default.json") as handle:
+    def get_file_mocked(self, x_system, file_id):
```
Comment (mr-kamran-ali) [inline tests/drop/mocked_drop.py:34]:
```suggestion
    def get_file_mocked(self, x_system:str, file_id:str):
```

**Review by mr-kamran-ali — COMMENTED** [review]
Look good, just few quick tiny fixes.

**Comment by erichesse** [comment]
> Look good, just few quick tiny fixes.

done, just had to lint on top of your suggestions

File: CHANGELOG.md [hunk 3]
```diff
@@ -33,12 +33,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - transform grippeweb resources
 - wikidata aux extractor into seq-repo
 - function `get_merged_organization_id_by_query_with_transform_and_load` to
-  wikidata.extract module
+ - wikidata.extract module
```
Comment (mr-kamran-ali) [inline CHANGELOG.md:36]: can you also remove a space in the beginning of line 36 please? it'll look better.

Comment (erichesse) [reply CHANGELOG.md]: https://github.com/robert-koch-institut/mex-extractors/pull/97/commits/23ea0e6a1005f2b4e123e5e4892cc4b913408da1

**Review by mr-kamran-ali — APPROVED** [review]
🚀

**Pull request merged**